libkbfs/test_stallers.go

```diff
@@ -311,75 +309,74 @@ func runWithContextCheck(ctx context.Context, action func(ctx context.Context) e
 	return err
 }
 
-// stallingBlockOps is an implementation of BlockOps whose operations
-// sometimes stall. In particular, if the operation name matches
-// stallOpName, and ctx.Value(stallKey) is a key in the corresponding
-// staller is used to stall the operation.
-type stallingBlockOps struct {
+// stallingBlockServer is an implementation of BlockServer whose
+// operations sometimes stall. In particular, if the operation name
+// matches stallOpName, and ctx.Value(stallKey) is a key in the
+// corresponding staller is used to stall the operation.
+type stallingBlockServer struct {
 	stallOpName StallableBlockOp
 	// stallKey is a key for switching on/off stalling. If it's present in ctx,
 	// and equal to `true`, the operation is stalled. This allows us to use the
 	// ctx to control stallings
 	stallKey string
 	staller  staller
-	delegate BlockOps
+	delegate BlockServer
```

Comment:
Since you're passing through most functions now, may be better to just embed `BlockServer` and remove the delegating functions. What do you think?